apps/etterlatte-saksbehandling-ui/client/src/components/behandling/sjekkliste/Sjekkliste.tsx

```diff
@@ -233,6 +234,14 @@ const SjekklisteItem = ({
         }}
       >
         {item.beskrivelse}
+        {item.beskrivelse.startsWith(utbetaltBidragsforskudd) && (
+          <>
+            <br />
+            <Link href={configContext['bisysUrl']} target="_blank">
+              Bisys <ExternalLinkIcon />
+            </Link>
+          </>
+        )}
```

Comment:
😬 ... Ikke helt fan av denne løsningen. 

Burde sjekkpunkter heller hatt støtte for lenke? 
Eventuelt at de er HTML sånn at lenken kan være en del av teksten?

Reply:
Jepp endret til å vises uansett på BP under sjekkpunktene.